### ISSUE TITLE
Plugin broken under Mac OSX

### ISSUE THREAD

**Issue by EckartHilliger** [kiwigrid/helm-maven-plugin#14]
Version: 1.5

Reproduce:
change config (pom.xml) from 
```
<properties>
    <helm.download.url>https://kubernetes-helm.storage.googleapis.com/helm-v2.6.1-linux-amd64.tar.gz</helm.download.url>
    <helm.repo.url>https://chartmuseum.infra.nanogridcloud.com</helm.repo.url>
</properties>
```
to
```
<properties>
    <helm.download.url>https://kubernetes-helm.storage.googleapis.com/helm-v2.6.1-darwin-amd64.tar.gz</helm.download.url>
    <helm.repo.url>https://chartmuseum.infra.nanogridcloud.com</helm.repo.url>
</properties>
```
then run ``mvn package``. The download of helm is working properly, and a ``darwin-amd`` folder is created under ``target/helm``
When trying to invoke helm, however, it tries to start in folder ``linux-amd`` which doesn't exist.

Workaround:
create a symlink from ``darwin-amd`` to ``linux-amd`` under ``target/helm``

**Comment by mschiffel** [comment]
You can add the following profile settings to you `pom.xml`:
```
  <profiles>
		<profile>
			<id>linux-profile</id>
			<activation>
				<os>
					<family>unix</family>
				</os>
			</activation>
			<properties>
				<helm.download.url>https://kubernetes-helm.storage.googleapis.com/helm-v2.7.2-linux-amd64.tar.gz</helm.download.url>
			</properties>
		</profile>

		<profile>
			<id>mac-profile</id>
			<activation>
				<os>
					<family>mac</family>
				</os>
			</activation>
			<properties>
				<helm.download.url>https://kubernetes-helm.storage.googleapis.com/helm-v2.7.2-darwin-amd64.tar.gz</helm.download.url>
			</properties>
		</profile>
	</profiles>
```

**Comment by axdotl** [comment]
Fixed by #45
